docs/src/componentDocs/ScoreCard/playground/PreviewComponent.tsx

```diff
@@ -0,0 +1,210 @@
+import { Hero } from '@brightlayer-ui/react-components/core/Hero';
+import { HeroBanner } from '@brightlayer-ui/react-components/core/HeroBanner';
+import Cloud from '@mui/icons-material/Cloud';
+import Favorite from '@mui/icons-material/Favorite';
+import Humidity from '@brightlayer-ui/icons-mui/Moisture';
+import List from '@mui/material/List';
+import ListAlt from '@mui/icons-material/ListAlt';
+import Mail from '@mui/icons-material/Mail';
+import MoreVert from '@mui/icons-material/MoreVert';
+import Notifications from '@mui/icons-material/Notifications';
+import Search from '@mui/icons-material/Search';
+import Temp from '@brightlayer-ui/icons-mui/Temp';
+import { createProps, getImage, hideDefaultPropsFromSnippet, removeEmptyLines } from '../../../shared/utilities';
+import { PropsType } from '../../../__types__';
+import PreviewComponentWithCode from '../../../shared/PreviewComponentWithCode';
+
+import * as Colors from '@brightlayer-ui/colors';
+
+export const PreviewComponent = (): JSX.Element => {
+    const scoreCardJson = useAppSelector((state: RootState) => state.componentsPropsState.scoreCardComponent);
+
+    const scoreCardProps = createProps(scoreCardJson.props as PropsType[]);
+    const scoreCardOtherProps = createProps(scoreCardJson.otherProps as PropsType[]);
+
+    const toggleDefaultProp = (propName: string, currentValue: any): string =>
+        hideDefaultPropsFromSnippet(scoreCardJson, propName, currentValue, 'props');
+
+    const actionItems = [
+        <Search key={'search'} />,
+        <Mail key={'mail'} />,
+        <Notifications key={'notifications'} />,
+        <Favorite key={'favorite'} />,
+        <Cloud key={'cloud'} />,
+        <MoreVert key={'morevert'} />,
+    ];
+
+    const heroes: JSX.Element[] = [
+        <Hero
+            key={'hero1'}
+            icon={<Temp fontSize={'inherit'} htmlColor={Colors.black[500]} />}
+            label={'Temperature'}
+            iconSize={48}
+            iconBackgroundColor={Colors.white[50]}
+            ChannelValueProps={{ value: 98, units: '°F' }}
+            fontSize={'normal'}
+        />,
+        <Hero
+            key={'hero2'}
+            icon={<Humidity fontSize={'inherit'} htmlColor={Colors.blue[300]} />}
+            label={'Humidity'}
+            iconBackgroundColor={Colors.white[50]}
+            ChannelValueProps={{ value: 54, units: '%' }}
+            iconSize={48}
+            fontSize={'normal'}
+        />,
+    ];
+
+    const showHeroSection = (noOfHeroes: number): string => {
+        switch (noOfHeroes) {
+            case 0:
+                return '';
+            case 1:
+                return `badge={
+            <HeroBanner>
+                <Hero
+                    key={'hero1'}
+                    icon={<Temp fontSize={'inherit'} htmlColor={Colors.black[500]} />}
+                    label={'Temperature'}
+                    iconSize={48}
+                    iconBackgroundColor={Colors.white[50]}
```

Comment:
This might be a stretch goal, but is it possible to pass in `'background.paper'` here to take advantage of the MUIv5 sx props, instead of `Colors.white[50]`? I'd love to see imported constants like `Colors` used less frequently in the playground code snippet.

Reply:
I have added `sx` props but code snippet style starts breaking.
<img width="892" alt="Screenshot 2022-09-20 at 3 07 26 PM" src="https://user-images.githubusercontent.com/10433274/191223976-c0166223-0af4-45c9-89a7-0dd8a360ff09.png">

Reply:
> Starting to add variables with references will make the code snippets only work partially, I'd stick to hardcoding the colors like you've been doing Ekta.

reverted sx prop changes and using hardcoded color.